tests/conftest.py

```diff
@@ -183,323 +177,112 @@ def dummy_input_bundle():
     return InputBundle([])
 
 
-# TODO: remove me, this is just string.encode("utf-8").ljust()
-# only used in test_logging.py.
-@pytest.fixture
-def bytes_helper():
-    def bytes_helper(str, length):
-        return bytes(str, "utf-8") + bytearray(length - len(str))
-
-    return bytes_helper
-
-
-def _none_addr(datatype, data):
-    if datatype == "address" and int(data, base=16) == 0:
-        return (datatype, None)
-    else:
-        return (datatype, data)
-
-
-CONCISE_NORMALIZERS = (_none_addr,)
-
-
 @pytest.fixture(scope="module")
-def tester():
+def gas_limit():
     # set absurdly high gas limit so that london basefee never adjusts
-    # (note: 2**63 - 1 is max that evm allows)
-    custom_genesis = PyEVMBackend._generate_genesis_params(overrides={"gas_limit": 10**10})
-    custom_genesis["base_fee_per_gas"] = 0
-    backend = PyEVMBackend(genesis_parameters=custom_genesis)
-    return EthereumTester(backend=backend)
-
-
-def zero_gas_price_strategy(web3, transaction_params=None):
-    return 0  # zero gas price makes testing simpler.
+    # (note: 2**63 - 1 is max that py-evm allows)
+    return 10**10
 
 
 @pytest.fixture(scope="module")
-def w3(tester):
-    w3 = Web3(EthereumTesterProvider(tester))
-    w3.eth.set_gas_price_strategy(zero_gas_price_strategy)
-    return w3
-
-
-def get_compiler_gas_estimate(code, func):
-    sigs = compiler.phases.CompilerData(code).function_signatures
-    if func:
-        return compiler.utils.build_gas_estimates(sigs)[func] + 22000
-    else:
-        return sum(compiler.utils.build_gas_estimates(sigs).values()) + 22000
-
-
-def check_gas_on_chain(w3, tester, code, func=None, res=None):
-    gas_estimate = get_compiler_gas_estimate(code, func)
-    gas_actual = tester.get_block_by_number("latest")["gas_used"]
-    # Computed upper bound on the gas consumption should
-    # be greater than or equal to the amount of gas used
-    if gas_estimate < gas_actual:
-        raise Exception(f"Gas upper bound fail: bound {gas_estimate} actual {gas_actual}")
-
-    print(f"Function name: {func} - Gas estimate {gas_estimate}, Actual: {gas_actual}")
-
-
-def gas_estimation_decorator(w3, tester, fn, source_code, func):
-    def decorator(*args, **kwargs):
-        @wraps(fn)
-        def decorated_function(*args, **kwargs):
-            result = fn(*args, **kwargs)
-            if "transact" in kwargs:
-                check_gas_on_chain(w3, tester, source_code, func, res=result)
-            return result
-
-        return decorated_function(*args, **kwargs)
-
-    return decorator
-
-
-def set_decorator_to_contract_function(w3, tester, contract, source_code, func):
-    func_definition = getattr(contract, func)
-    func_with_decorator = gas_estimation_decorator(w3, tester, func_definition, source_code, func)
-    setattr(contract, func, func_with_decorator)
+def initial_balance():
+    return 0  # default balance for the deployer account
```

Comment:
i'm not sure why this is necessary?

Reply:
it was an easy way to make sure there are enough funds for tests sending value, but it can be easily done inside the tests